lib/spack/spack/installer.py

```diff
@@ -693,6 +699,17 @@ def __str__(self):
         return '{0}: {1}; {2}; {3}; {4}'.format(
             self.pid, requests, tasks, installed, failed)
 
+    def _set_term_title(self, text, force=False):
```

Comment:
The `force` isn't used by any caller. Was it meant for unit tests? (I don't see any actually.)

Reply:
That's actually just something I forgot to remove when updating the PR in the past. It's not necessary anymore.